### PR TITLE
Nicer name for alarms

### PR DESCRIPTION
**Pull request by rasmusbe**
Changes to better names for entities

### CODE REVIEW

**Comment by AaronDavidSchneider** [comment]
Hi, I like the idea! Thanks for the PR. 
We should make sure that the name also changes when the attributes change.

File: custom_components/sonos_alarm/switch.py [hunk 1]
```diff
@@ -131,6 +128,13 @@ def __init__(self, soco, alarm):
             ATTR_PLAY_MODE: str(self.alarm.play_mode),
             ATTR_SCHEDULED_TODAY: self._is_today
         }
+
+        self._name = "Sonos Alarm {} {} {}".format(
+            speaker_info["zone_name"],
+            self.alarm.recurrence.title(),
+            str(self.alarm.start_time)[0:5]
+        )
+
```
Comment (AaronDavidSchneider) [inline custom_components/sonos_alarm/switch.py:137]: This should be easily achievable if you include those lines also in the update routine.

**Review by AaronDavidSchneider — CHANGES_REQUESTED** [review]

File: custom_components/sonos_alarm/switch.py (hunk 1 above)
Comment (rasmusbe) [reply custom_components/sonos_alarm/switch.py:137]: You are right, I've added that now!

**Review by AaronDavidSchneider — APPROVED** [review]
Looks good!

**Pull request merged**